### DISCUSSION TITLE
Fits to Slater-type functions

### DISCUSSION THREAD

**Discussion by PaulWAyers** [theochem/BFit#33] [Q&A] [answered]
@Ali-Tehrani do we support fitting to Slater functions? I know expanding in Gaussians is done, but based on today's discussions with @JamesSMAnderson, it may be useful to fit to Slaters also.

**Comment by Ali-Tehrani** [comment, marked as answer]
It wouldn't be hard to implement it for either SLSQP and (I think) the fixed-point method. I'll try to do it soon
